src/frontends/pytorch/src/transforms/append_list_unpack_replacer.cpp

```diff
@@ -62,11 +62,12 @@ AppendListUnpackReplacer::AppendListUnpackReplacer() {
             auto getitem_index_ptr = getitem_node->input_value(1).get_node_shared_ptr();
             auto getitem_index_const = std::dynamic_pointer_cast<ov::op::v0::Constant>(getitem_index_ptr);
             auto index_val = getitem_index_const->cast_vector<int64_t>();
-            auto index = 0;
-            if (index_val[0] >= 0) {
-                index = index_val[0];
-            } else {
-                index = inputs.size() + index_val[0];
+            if (index_val.size() != 1) {
+                return false;
+            }
+            auto index = index_val[0];
+            if (index_val[0] < 0) {
+                index = inputs.size() + index;
```

Comment:
here we try to sum-up `size_t` and `int64_t` element. Please check

Reply:
If compiler doesn't output warnings I think it is cleaner to not use extra casts, @ilya-lavrenov what do you think?

Reply:
What is fine for one compiler can be "death" for another one:)